test/Index/objc_conformances.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: %target-swift-ide-test -print-indexed-symbols -source-filename %s
+
+// Let's just make sure this doesn't crash for now; there's nothing interesting
+// to see in the output here.
```

Comment:
Nitpick: validation-test, then?

Reply:
If you add at least a single CHECK line it will be useful to serve as a sanity check that AST walking did take place.

Reply:
I’ll add a non-optional requirement and check that the witness got walked.